doc/theory/glossary.tex

```diff
@@ -1,12 +1,64 @@
+	\begin{equation*}
+	d_i[v_0, \dots, v_n] = [v_0, \dots, \widehat{v}_i, \dots, v_n].
+	\end{equation*}
+	A \textbf{simplicial complex} $X$ is a finite union of simplices in $\mathbb{R}^N$ satisfying that every face of a simplex in $X$ is in $X$ and that the non-empty intersection of two simplices in $X$ is a face of each. A \textbf{subcomplex} of $X$ is a simplicial complex whose simplices are also simplices of $X$. We denote the set of $n$-dimensional simplices by $X_n$.
+	
+	\subsection*{Cubical complex}
+	
+	An \textbf{elementary interval} $I_a$ is a subset of $\mathbb{R}$ of the form $[a, a+1]$ or $[a,a] = \{a\}$ for some $a \in \mathbb{R}$. These two types are called respectively \textbf{non-degenerate} and \textbf{degenerate}. To a non-degenerate elementary interval we assign two degenerate elementary intervals
+	\begin{equation*}
+	d^+ I_a = [a+1, a+1] \qquad \text{and} \qquad d^- I_a = [a, a].
+	\end{equation*}
+	An \textbf{elementary cube} is a subset of the form 
+	\begin{equation*}
+	I_{a_1} \times \cdots \times I_{a_N} \subset \mathbb{R}^N
+	\end{equation*}
+	where each $I_{a_i}$ is an elementary interval. We refer to the total number of its non-degenerate factors $I_{a_{k_1}}, \dots, I_{a_{k_n}}$ as its \textbf{dimension} and, assuming
+	\begin{equation*}
+	a_{k_1} < \cdots < a_{k_{n,}}
+	\end{equation*}
+	we define for $i = 1, \dots, n$ the following two elementary cubes
+	\begin{equation*}
+	d_i^\pm I^N = I_{a_1} \times \cdots \times d^\pm I_{a_{k_i}} \times \cdots \times I_{a_{N.}}
+	\end{equation*}
+	
+	A \textbf{cubical complex} is a finite set of elementary cubes of $\mathbb{R}^N$, and a \textbf{subcomplex} of $X$ is a cubical complex whose elementary cubes are also in $X$. We denote the set of $n$-dimensional cubes as $X_n$.
+	
+	\paragraph{Reference:} \cite{mischaikow04computational}
 	
-	A \textit{simplicial complex} is a pair of sets $(V, X)$ with the elements of $X$ being subsets of $V$ such that: 1) For every $v$ in $V$, the singleton $\{v\}$ is in $X$. 2) If $x$ is in $X$ and $y$ is a subset of $x$, then $y$ is in $X$. We abuse notation and denote the pair $(V,X)$ simply by $X$.
+	\subsection*{Abstract simplicial complex:} \label{simplicial complex}
+	
+	An \textit{abstract simplicial complex} is a pair of sets $(V, X)$ with the elements of $X$ being subsets of $V$ such that: 
+	\begin{enumerate}
+		\item for every $v$ in $V$, the singleton $\{v\}$ is in $X$ and
+		\item if $x$ is in $X$ and $y$ is a subset of $x$, then $y$ is in $X$. 
+	\end{enumerate}
+	We abuse notation and denote the pair $(V, X)$ simply by $X$.
 	
 	The elements of $X$ are called \textit{simplices} and the \textit{dimension} of a simplex $x$ is defined by $|x| = \# x - 1$ where $\# x$ denotes the cardinality of $x$. Simplices of dimension $d$ are called $d$-simplices. We abuse terminology and refer to the elements of $V$ and to their associated $0$-simplices both as \textit{vertices}.
```

Comment:
didn't you mean `abstract simplicial complex` (instead of simplicial complex) below?
Also, it seems a bit redundant with the definition in simplex - is it possible to put the definitions of k-skeleton and subcomplex in `simplicial complex` (only?).